services/web-server/src/loaders/artifacts.js

```diff
@@ -1,5 +1,5 @@
 const DataLoader = require('dataloader');
-const sift = require('sift').default;
+const siftUtil = require('../utils/siftUtil');
```

Comment:
We can rename the utility to `sift`:

```jsx
const sift = require('../utils/sift');
```

This comment applies to all the other places we're importing this utility.